sys/shell/commands/sc_lwip_netif.c

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
A better way would be to implement the missing functionality of the `netif` API in lwIP, finish the port of GNRC's `ifconfig` to `netif` (so remove all usage of `gnrc_*` there and just use the same `ifconfig` command for both stacks. But as this is not the case right now, I am fine with this interim solution for now.

Reply:
That makes sense. GNRC and its netif felt closely connected so I didn't dare to attempt it now. I might try something like it for the ping command.

Reply:
Yes, when I rewrote the ping command to be asynchronous, there was no `sock_async` and no `sock_*_recv_aux` yet, so it might be possible to implement `ping` now fully featured using only `sock_ip` now.